Tests/Packet++Test/Tests/IPv4Tests.cpp

```diff
@@ -1,16 +1,16 @@
 #include "../TestDefinition.h"
 #include "../Utils/TestUtils.h"
-#include <sstream>
 #include "EndianPortable.h"
-#include "Logger.h"
-#include "MacAddress.h"
-#include "Packet.h"
 #include "EthLayer.h"
 #include "IPv4Layer.h"
 #include "IPv6Layer.h"
-#include "UdpLayer.h"
+#include "Logger.h"
+#include "MacAddress.h"
+#include "Packet.h"
 #include "PayloadLayer.h"
 #include "SystemUtils.h"
+#include "UdpLayer.h"
+#include <sstream>
```

Comment:
why does it change the order of the `#include` files? 🤔

Reply:
For some of the files, I ran the formatting through VSCode formatter, which resulted in the reordering. Can we keep the formatted version as long as it doesn't break the code?

Reply:
is it possible to revert? I'd like to keep the changes as minimal and consistent as possible. Also, why is VSCode formatter chaging the include order? Doesn't it have the same rules defined [here](https://github.com/seladb/PcapPlusPlus/blob/master/.clang-format)?

Reply:
sure, let me revert. It seems that VSCode formatter sometimes doesn't listen to the config lol.

Reply:
@tigercosmos this comment was not addressed yet...

Reply:
I missed it, just fixed.